oci/oci_unix.go

```diff
@@ -60,6 +60,10 @@ func genericCalculateCPUPercent(cpuTotal uint64, perCPU []uint64) float64 {
 	return cpuPercent
 }
 
+func setSize(fd uintptr, size remotecommand.TerminalSize) error {
+	return term.SetWinsize(fd, &term.Winsize{Height: size.Height, Width: size.Width})
```

Comment:
do you need term here?

Reply:
Yes, that's how the removed K8s looked like. I wanted to avoid changing that in hope to not introduce a regression.